frontend/app/tracking/ActivityTracking.scala

```diff
@@ -101,6 +101,7 @@ case class EventData(event: RichEvent) {
 
   val group = event match {
```

Comment:
This (already existing) `match` statement is a little sad, you would have thought these strings could live within the Event class, or better, into the refactored EventProvider class that I keep talking about but never writing. Can let it go for the moment.